_posts/databases/postgresql/timescaledb/2000-01-01-intro.md

```diff
@@ -0,0 +1,20 @@
+---
+title: TimescaleDB
+nav: Introduction
+modified_at: 2022-01-21 00:00:00
+tags: timescale databases postgresql extensions
+index: 1
+---
+
+TimescaleDB is a PostgreSQL extension for the support of the time-series data.
+This extension add several features and functions, you can find more information
+on their documentation: [https://docs.timescale.com/api/latest](https://docs.timescale.com/api/latest).
```

Comment:
I find it ugly to have the URL displayed, I prefer a link with a meaningful text
```suggestion
TimescaleDB is a PostgreSQL extension for the support of time-series data.
This extension adds several features and functions. You can find more information
on the [official documentation](https://docs.timescale.com/api/latest).
```